libr/core/cbin.c

```diff
@@ -1545,7 +1545,7 @@ static int bin_relocs(RCore *r, int mode, int va) {
 			if (bin_demangle) {
 				char *mn = r_bin_demangle (r->bin->cur, NULL, name, addr);
 				if (mn && *mn) {
-					free (name);
+					R_FREE (name);
```

Comment:
this is unnecesary because name is assigned right after being freed. so the call to `free` is enough

Reply:
Yes you re right for this one.